tests/unit/test_unit_line.py

```diff
@@ -0,0 +1,32 @@
+"""Line unit tests."""
+
+from unittest.mock import patch
+
+from lantmateriet.line import Line
+
+
+class TestUnitLine:
```

Comment:
general question considering the tests, how do you run the test locally? I would do like it like letting TestUnitLine be a subclass of Testcase (from unittest import TestCase), and run unittest.main().

Reply:
From the repo root run

```bash
pip install -e ".[dev]"
```

and then it should be enough to run

```bash
pytest
```

or if you want to just run the unit tests

```bash
pytest tests/unit
```

Reply:
ok, perfect